featuretools/tests/entityset_tests/test_es.py

```diff
@@ -934,3 +934,23 @@ def test_sizeof(entityset):
         total_size += entity.last_time_index.__sizeof__()
 
     assert entityset.__sizeof__() == total_size
+
+
+def test_construct_without_id():
+    assert ft.EntitySet().id is None
+
+
+def test_repr_without_id():
+    match = 'Entityset: None\n  Entities:\n  Relationships:\n    No relationships'
+    assert repr(ft.EntitySet()) == match
+
+
+def test_getitem_without_id():
+    error_text = 'Entity test does not exist in None'
```

Comment:
Let change this error text to not even reference `self.id`. So it can just say `'Entity test does not exist'`